src/components/form/fields/passwordPolicyRules.jsx

```diff
@@ -65,9 +65,17 @@ const PasswordPolicyIntro = withI18n(withTheme(styled(({ i18n, className }) =>
     margin-bottom: ${props => props.theme.get('spacing')}px;
 `));
 
-export const PasswordPolicyRules = withTheme(styled(({ className, ...props }) =>
+export const PasswordPolicyRules = withI18n(withTheme(styled(({ className, i18n, ...props }) =>
     <div className={className}>
+        {props.minStrength !== 0 && <PasswordPolicyRule>
```

Comment:
'{props.minStrength &&' should be enough to assert the condition.

Reply:
In fact, I don't want to display the check rule if the strength is set to "None". Your suggestion doesn't do that :)

Reply:
Edited yes my bad!

Reply:
Thanks for the suggestion anyway :)